x-pack/plugins/uptime/public/components/common/header/action_menu_content.test.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { fireEvent, waitFor } from '@testing-library/react';
+import { render } from '../../../lib/helper/rtl_helpers';
+import { ActionMenuContent } from './action_menu_content';
+
+describe('ActionMenuContent', () => {
+  it('renders alerts dropdown', async () => {
+    const { getByLabelText, getByText } = render(<ActionMenuContent />);
+
+    const alertsDropdown = getByLabelText('Open alert context menu');
+    fireEvent.click(alertsDropdown);
+
+    await waitFor(() => {
+      expect(getByText('Create alert'));
+      expect(getByText('Manage alerts'));
+    });
+  });
+
+  it('renders settings link', () => {
+    const { getByRole, getByText } = render(<ActionMenuContent />);
+
+    const settingsAnchor = getByRole('link', { name: 'Navigate to the Uptime settings page' });
```

Comment:
i usually avoid `getByRole` they tends to be slow.

Reply:
Was added as a response to [feedback](https://github.com/elastic/kibana/pull/100298#discussion_r637940414).